sdk-api-src/content/handleapi/nf-handleapi-duplicatehandle.md

```diff
@@ -332,7 +334,7 @@ Normally the target process closes a duplicated handle when that process is fini
 <ul>
 <li>Set <i>hSourceProcessHandle</i> to the target process from the <b>DuplicateHandle</b> call that created the handle.</li>
 <li>Set <i>hSourceHandle</i> to the duplicated handle to close.</li>
-<li>Set <i>lpTargetHandle</i> to <b>NULL</b>.</li>
+<li>Set <i>hTargetProcessHandle</i> to <b>NULL</b>.</li>
```

Comment:
`hTargetProcessHandle` is the important parameter to be set to **NULL**. `lpTargetHandle` doesn't have to be mentioned once it's documented that it's ignored in this case.